### ISSUE TITLE
Erro de Compilação (Arquivos faltando: Point,Robot)

### ISSUE THREAD

**Issue by heitorrapela** [VSS-SDK/VSS-Vision#64]
Olá, primeiramente gostaria de parabenizar pelo projeto. Estou tentando compilar e tive alguns problemas porque acredito que está faltando alguns arquivos.

![image](https://user-images.githubusercontent.com/14017613/63876958-f5f3fd80-c99c-11e9-8716-fd40eba7193c.png)

Eu tive este problema, ai acreditei que o vss::Point era apenas um wrapper para ponto, ai eu troquei no código todo para cv::Point.

Agora estou tendo um erro com o arquivo robot.h, ai achei melhor abrir a issue :)

![image](https://user-images.githubusercontent.com/14017613/63877021-19b74380-c99d-11e9-930a-31a8a8cd34be.png)

**Comment by johnfercher** [comment]
Parece que o VSS-Core não foi instalado. Você seguiu esse [tutorial](https://vss-sdk.github.io/book/install.html) de instalação?

Qual o seu sistema operacional? poderia rodar um lsb_release -a e informar a saida desse comando?

**Comment by heitorrapela** [comment]
Certo, me parece que este é o problema. Desculpa incomodar kkkk, vou tentar seguir direitinho, e se der certo, fecho a issue :)

**Comment by johnfercher** [comment]
Show de bola, se o erro persistir é só falar.

**Comment by heitorrapela** [comment]
@johnfercher esse erro resolveu, mas agora estou com outro. Meu protobuffer está na versão 3.4, e alguma coisa no projeto foi compilada com 2.6.1, não sei se foi o opencv, vou dar uma investigada.

![image](https://user-images.githubusercontent.com/14017613/64032903-c6223280-cb21-11e9-9aee-4f5270f39de3.png)

**Comment by johnfercher** [comment]
Isso não faz muito sentido, ele deveria usar a mesma versão que está instalada na sua máquina. Já tentou deletar essa pasta build e rodar o configure do projeto de novo?
